raiwidgets/raiwidgets/error_analysis_dashboard.py

```diff
@@ -73,15 +73,50 @@ class ErrorAnalysisDashboard(Dashboard):
     :param num_leaves: The number of leaves of the surrogate tree
         trained on errors.
     :type num_leaves: int
+    :param sample_dataset: Used only if explanation is not specified for
+        the dataset explorer.  Specify less than 10k points for optimal
+        performance.
+    :type sample_dataset: pd.DataFrame or numpy.ndarray or list[][]
+
+    :Example:
```

Comment:
Why is this both here and below? Shouldn't once be enough?

Reply:
I thought one was needed for sphinx and the other for help():
https://erroranalysis.ai/raiwidgets.html
but it looks like the sphinx is fine and help is fine.  I'm not sure how you made the sphinx use the top-level class doc instead of the init method doc.  Maybe it's only a problem in the azureml-sdk documentation.  Let me remove this then.

Reply:
Removed the doc below.